eth_typing/abi.py

```diff
@@ -17,34 +17,17 @@
 """Binary data to be decoded."""
 
 
-class ABIEventComponent(TypedDict, total=False):
+class ABIParam(TypedDict, total=False):
```

Comment:
I think I prefer `ABIComponent` here? Thoughts on that? It feels a bit more all-encompassing. I could go either way by my other comments will keep this as the thought. We can change those accordingly if we want to stick with param.

Reply:
The terminology can get blurry, `ABIComponent` is fine with me. I'll make sure the docs reflect that across our libs too.

Reply:
Yeah agreed. Not a strong opinion by any means. Just a lean 😅